src/main/java/edu/indiana/dlib/amppd/web/HttpComponentsClientHttpRequestFactoryBasicAuth.java

```diff
@@ -12,6 +12,8 @@
 import org.springframework.http.HttpMethod;
 import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
 
+// TODO This class is not used anymore and can probably be removed.
```

Comment:
Same as others.

Reply:
we might want to keep this one in case we impl anything for AMP rest api using basic auth. i think it's ok to decide this later.